GCRCatalogs/alphaq.py

```diff
@@ -110,21 +110,30 @@ def _collect_native_quantities(name, obj):
             'sersic_disk':              'morphology/diskSersicIndex',
             'sersic_bulge':             'morphology/spheroidSersicIndex',
             'ellipticity_true':         'morphology/totalEllipticity',
-            'ellipticity_disk_true':    'morphology/diskEllipticity',
```

Comment:
We should probably leave these alone for now? I am not sure if we have reached agreement on this... and we should also check if anyone is using them

Reply:
I'll put back disk/bulge ellip, ellip1/2.